test/dotnet-new.Tests/GivenThatIWantANewCSApp.cs

```diff
@@ -44,13 +43,13 @@ public void When_dotnet_build_is_invoked_Then_project_builds_without_warnings()
                 .Execute("new");
 
             new TestCommand("dotnet") { WorkingDirectory = rootPath }
-                .Execute("restore");
+                .Execute("restore3 -s https://dotnet.myget.org/F/cli-deps/api/v3/index.json");
```

Comment:
This is only in place to enable tests until we get the packages published to nuget.org. The `cli-deps` feed is a private snapshot of `dotnet-core` and several other feeds which provide dependencies into the CLI. It is our 'prediction' of what nuget.org will look like. We used the source feeds at some point in the past but found that differences in retention policies were creating grief for the CLI engineering team.